docs/troubleshooting.md

```
@@ -0,0 +1,21 @@
+# Troubleshooting
+
+This guide covers common issues and their solutions when working with MeetingMind.
+
+
+### Migration Errors
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)